flipperdevices/flipperzero-good-faps
# Picopass: check if HID CSN if read fails

# What's new

![Screenshot-20231013-173216](https://github.com/flipperdevices/flipperzero-good-faps/assets/115752/69d04eb3-7f54-4870-a360-3dd6fc8777e4)

# Verification 

- Find a picopass card that is not iClass
- Run Read Card
- See "Read Failed" and "Non-HID CSN"

# Checklist (For Reviewer)
